preflight/serve.py

```diff
@@ -0,0 +1,6 @@
+import cli
+
+@cli.preflight_plugin("serve",
+                      [cli.Argument("--port", "-p", type=int, default=8000)])
```

Comment:
In an ideal world there'd be a magic new type (`port`?) which could be passed into the command argument and would be magically handled without needing to use the preflight plugin.

There _may_ be a need for the idea of preflight plugins that could explicitly manipulate the docker container (e.g., little resident server which handles opening a URL, etc), but that seems like overkill right now.

Reply:
I think I might have been getting carried away with this suggestion. It's sufficient to have to write a little code to make this work right now; it can always be simplified in the future.